roles/install_toggle/tasks/recore.yml

```diff
@@ -76,3 +76,16 @@
 - name: enable the toggle service
   shell: systemctl enable toggle
 
+- name: Recore - copy toggle-runfirst service file
+  copy:
+    src: '{{ role_path }}/files/toggle-runfirst.service'
+    dest: /etc/systemd/system/toggle-runfirst.service
+
+- name: Recore - Script - Ensure ethernet comes up
+  copy:
+    src: '{{ role_path }}/files/toggle-runfirst'
+    dest: /usr/lib/
+    mode: +x
+
+- name: enable the toggle-runfirst service
+  shell: systemctl enable toggle-runfirst
```

Comment:
You can use the systemd ansible module. You'll need to specify that it also runs the daemon-reload, or out won't pick up the new systemd services you added.

Reply:
We have discussed this before, the systemd module does not seem to work when running the INSTALL-*.yml scripts. I would like to use the systemd module, but as far as I can tell there is a bug causing it to not work for me.

Reply:
Maybe there is a way to separate the two scenarios? I understand that you probably don't want to spend time investigating this, but running INSTALL-toggle.yml behaves differently than build-image-in-chroot-end-to-end.sh

Reply:
Yes, and they should.

The ansible scripts we want to use during image build are the SYSTEM-*.yml

Reply:
The install scripts are meant to run off you already run on the final board...

Reply:
I've added a rescue block, we can try that.